islandora_internet_archive_bookreader.module

```diff
@@ -57,6 +65,7 @@ function islandora_internet_archive_bookreader_theme() {
       'variables' => array(
         'object' => NULL,
         'datastream_id' => 'JP2',
+        'backup_datastream_id' => 'JPG',
```

Comment:
Actually, it's pretty unlikely (due bad theme hook implementation names)... I dunno...